server/features/content_rewrite.feature

```diff
@@ -0,0 +1,485 @@
+Feature: Rewrite content
+
+    @auth
```

Comment:
I modified the first test to check if the rewritten story goes to same desk and incoming stage.